image_generation/stable_diffusion_1_5/cpp/README.md

```diff
@@ -71,17 +71,18 @@ cmake --build build --parallel
 
 ## Step 4: Run Pipeline
 ```shell
-./build/stable_diffusion [-p <posPrompt>] [-n <negPrompt>] [-s <seed>] [--height <output image>] [--width <output image>] [-d <device>] [-r <readNPLatent>] [-l <lora.safetensors>] [-a <alpha>] [-h <help>] [-m <modelPath>] [-t <modelType>] [--dynamic]
+./build/stable_diffusion [-p <posPrompt>] [-n <negPrompt>] [-s <seed>] [--height <output image>] [--width <output image>] [-d <device>] [-r <readNPLatent>] [-l <lora.safetensors>] [-a <alpha>] [-h <help>] [-m <modelPath>] [-t <modelType>] [--guidanceScale <guidanceScale>] [--dynamic]
 
 Usage:
   stable_diffusion [OPTION...]
 ```
 
-* `-p, --posPrompt arg` Initial positive prompt for SD  (default: cyberpunk cityscape like Tokyo New York  with tall buildings at dusk golden hour cinematic lighting)
-* `-n, --negPrompt arg` Default is empty with space (default: )
+* `-p, --posPrompt arg` Initial positive prompt for SD (default: cyberpunk cityscape like Tokyo New York  with tall buildings at dusk golden hour cinematic lighting)
+* `-n, --negPrompt arg` The prompt **not** to guide the image generation. Ignored when not using guidance (`--guidanceScale` is less than `1`) (default: )
```

Comment:
```suggestion
* `-n, --negPrompt arg` The prompt to guide the image generation away from. Ignored when not using guidance (`--guidanceScale` is less than `1`) (default: )
```
You should also align the description with main.cpp help

Reply:
Fixed